ReactTask/src/components/CreateAppointment/InputsAppointment.js

```diff
@@ -117,7 +127,13 @@ const InputsAppointment = ({ setFilledData, setDoctorChoose }) => {
       <StylBoxColumnFlexMargin dataReady={true}>
         <StylTitleVisitDoctor>Note</StylTitleVisitDoctor>
         <StylBoxColumnFlex>
-          <StylInputVisitDoctor placeholder='Leave a note if needed' />
+          <StylInputVisitDoctor
+            id='userNote'
+            role='userNote'
+            data-testid='userNote'
+            placeholder='Leave a note if needed'
+            onChange={(e) => setNoteUser(e.currentTarget.value)}
+          />
```

Comment:
I like the way you set values in this case. That's good

Reply:
![image](https://user-images.githubusercontent.com/48753830/144770792-45761cdc-b1ec-4d19-93c3-21a062a611d6.png)
Take a closer look
There is a click here